packages/@react-stately/layout/src/TableLayout.ts

```diff
@@ -336,6 +341,12 @@ export class TableLayout<T> extends ListLayout<T> {
       }
     }
 
+    // TODO: perhaps make a separate buildLoader? Do we need to differentiate the layoutInfo information?
+    // I think the below is ok for now since we can just treat nested loaders/load more as rows
+    if (node.type === 'loader') {
+      height = this.rowHeight;
+    }
```

Comment:
this is an assumption, we may want to open up the loader height to be configurable by the user

Reply:
I think someone was just asking for this somewhere, but I can't find where. Seems like something we should support

Reply:
would need to handle when row heights are estimated too